spec/views/bulk_ingest/show.html.erb_spec.rb

```diff
@@ -12,4 +12,33 @@
     expect(rendered).to have_selector "input[type='checkbox'][name='preserve_file_names'][value='1']"
     expect(rendered).to have_selector "select[name='rights_statement'] option[value='http://rightsstatements.org/vocab/CNE/1.0/']"
   end
+
+  describe "the page heading" do
+    context "when bulk ingesting a scanned resource" do
+      it "displays all kinds of things you could ingest" do
+        assign :resource_class, ScannedResource
+        assign :visibility, []
+        assign :states, []
+        assign :collections, []
+        render
+
+        # rendered is full of awkward line breaks and large spaces
+        xml = Nokogiri.parse(rendered)
+        heading_text = xml.css("h2").text.split(" ").join(" ")
+        expect(heading_text).to eq "Bulk Ingest Scanned Resources, Videos, or Vendor Bags"
```

Comment:
I think in view specs we have Capybara helpers. You should be able to do `expect(rendered).to have_css("h2", text: "Bulk Ingest Scanned Resources, Videos, or Vendor Bags"`

Reply:
ha, yay!